addons/website_event_exhibitor/views/event_exhibitor_templates_list.xml

```diff
@@ -31,33 +31,116 @@
 
 <!-- Main topbar -->
 <template id="exhibitors_topbar" name="Exhibitor Tools">
-    <nav class="navbar navbar-light border-top shadow-sm d-print-none">
-        <div class="container">
-            <div class="d-flex flex-column flex-sm-row justify-content-between w-100">
-                <form class="o_wevent_event_tags_form" action="#" method="POST">
-                    <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
-                    <ul class="o_wesponsor_topbar_filters o_wevent_index_topbar_filters nav"/>
-                </form>
-                <div class="o_wevent_event_search_box d-flex align-items-center flex-wrap ps-sm-3 pe-0">
-                    <t t-call="website_event_exhibitor.exhibitor_search_box">
-                        <t t-set="search" t-value="search_key" />
-                        <t t-set="placeholder" t-value="'Search an exhibitor ...'"/>
-                    </t>
+    <div class="d-flex d-print-none align-items-center justify-content-end flex-wrap gap-2 w-100">
+        <h3 class="my-0 me-auto pe-sm-4">Exhibitors</h3>
+        <form class="o_wevent_event_tags_form d-none d-lg-block" action="#" method="POST">
+            <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+            <div class="o_wesponsor_topbar_filters o_wevent_index_topbar_filters d-flex gap-2"/>
+        </form>
+        <div class="o_wevent_search d-flex w-100 w-lg-auto">
+            <div class="o_wevent_event_search_box d-flex align-items-center flex-wrap flex-grow-1">
+                <t t-call="website_event_exhibitor.exhibitor_search_box">
+                    <t t-set="search" t-value="search_key" />
+                    <t t-set="placeholder" t-value="'Search an exhibitor ...'"/>
+                </t>
+            </div>
+            <button class="btn btn-light position-relative ms-2 d-lg-none"
+                data-bs-toggle="offcanvas"
+                data-bs-target="#o_wevent_exhibitors_offcanvas">
+                <i class="fa fa-sliders"/>
+            </button>
+        </div>
+    </div>
+    <!-- Off canvas filters on mobile-->
+    <div id="o_wevent_exhibitors_offcanvas" class="o_website_offcanvas offcanvas offcanvas-end d-lg-none p-0 overflow-visible">
+        <div class="offcanvas-header">
+            <h5 class="offcanvas-title">Filters</h5>
+            <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"/>
+        </div>
+        <div class="offcanvas-body p-0">
+            <div class="accordion accordion-flush">
```

Comment:
Here's an idea. We can use the option templates so it's much less monolithic and we respect the options of the users.
See this here commit, which seems to work fine.
https://github.com/odoo-dev/odoo/commit/bc74708de04601f6cc1afb7e59e1236c9c0fb07e

On that note the options don't appear anymore because we used 'nav' as a selector ^^
Easy fix here, no need to use 'nav' there's only one of these. The options won't be available when using the editor on mobile.. but the editor on mobile doesn't really work anyway, and nobody would use it even if it did probably.
https://github.com/odoo-dev/odoo/commit/651a65fe11bf7f810f9a236dc7932cd5f9336e32